style.css

```diff
@@ -0,0 +1,86 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+header {
+  margin-top: 44px;
+  display: flex;
```

Comment:
[OPTIONAL] You can make `margin-top: 44px;` smaller to be exactly like the Figma design.